com.playeveryware.eos/Runtime/Core/Config/SandboxId.cs

```diff
@@ -70,6 +71,22 @@ readonly get
             }
         }
 
+        /// <summary>
+        /// Indicates whether the sandbox id is empty. A sandbox id is empty if
+        /// either the underlying value is null or empty, or if the underlying
+        /// value is equal to the string value for an empty Guid with the dashes
+        /// removed.
+        /// </summary>
+        [JsonIgnore]
+        public readonly bool IsEmpty
+        {
+            get
+            {
+                return String.IsNullOrEmpty(_value) || 
+                       Guid.Empty.ToString().Replace("-", "").Equals(_value);
```

Comment:
Instead of doing this - we should utilize the `.ToString("N")` function - since that does the appropriate stripping.

Reply:
Also look into whether this property can or should be utilized prior to applying the value to start up parameters of the EOS SDK.

Reply:
Property was updated to correctly use the `ToString("N")` function, and (since doing so would be outside the scope of the PR) a comment was added to `IsEmpty` suggesting that the function be utilized prior to being used to initialize the EOS SDK.

Marking conversation as resolved accordingly.